src/commonMain/kotlin/fr/acinq/lightning/crypto/LocalKeyManager.kt

```diff
@@ -11,6 +11,7 @@ import fr.acinq.lightning.transactions.Transactions
 data class LocalKeyManager(val seed: ByteVector, val chainHash: ByteVector32) : KeyManager {
 
     private val master = DeterministicWallet.generate(seed)
+    override val legacyNodeKey: DeterministicWallet.ExtendedPrivateKey = derivePrivateKey(master, eclairNodeKeyBasePath(chainHash))
```

Comment:
We should have a test to check that this key is correct (with test data generated from the legacy app).